### DISCUSSION TITLE
Request for COVID-19 Hospital Dataset for Research.

### DISCUSSION THREAD

**Discussion by nisso25** [owid/covid-19-data#2818] [General]
Hello ,I am currently working on a research project focusing on the impact of COVID-19 on hospital patients. I am in need of a specific dataset to support my research, but my efforts to locate a suitable dataset have been quite challenging. I am reaching out to this knowledgeable community for assistance.

Required Dataset Features:
I am looking for a comprehensive hospital dataset during the COVID-19 pandemic that includes the following features:

Age
Gender
Medical history/underlying health conditions
COVID-19 test results (positive/negative)
Results of respiratory rate analysis
Oxygen saturation levels
Blood pressure readings
Temperature measurements
Laboratory test results
Hospitalization history (whether the patient has been previously hospitalized for COVID-19 or not)
Purpose of the Dataset:

This dataset will be utilized to develop predictive models for identifying severe and non-severe COVID-19 cases among hospital patients. Understanding the factors that contribute to the severity of the disease.
Thank you in advance for your assistance,

**Comment by lucasrodes** [comment]
Dear @nisso25, thanks for your interest in this project.

Sorry, our hospitalisation metrics are not broken down by gender or age.

You can find all our hospitalisation data here: https://github.com/owid/covid-19-data/tree/master/public/data/hospitalizations
